tests/test_stats.py

```diff
@@ -1,201 +1,117 @@
 import json
 import uuid
+from lxml import html
+
```

Comment:
You don't need this blank line according to PEP8. https://www.python.org/dev/peps/pep-0008/#imports